cookiecutter/utils.py

```diff
@@ -107,7 +107,23 @@ def prompt_and_delete(path, no_input=False):
 
 
 def simple_filter(filter_function):
-    """Decorate a function to wrap it in a simplified jinja2 extension."""
+    """
+    Register a function for use as a filter extension for Jinja templates.
```

Comment:
I think flake8 is upset with this:

```suggestion
    """Register a function for use as a filter extension for Jinja templates.
```